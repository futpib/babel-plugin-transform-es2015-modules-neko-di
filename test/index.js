
const test = require('ava');

const babel = require('babel-core');

test('works', t => {
	const actual = babel.transform(`
		import defaultMember from "module-name";
		export default 42;
	`, {
		plugins: ['.'],
		moduleId: 'module-id'
	}).code;
	t.snapshot(actual); // eslint-disable-line ava/use-t-well
});
