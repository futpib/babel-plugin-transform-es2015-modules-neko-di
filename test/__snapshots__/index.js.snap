exports[`works 1`] = `"\"define(\\\"module-id\\\", [\\\"exports\\\", \\\"module-name\\\"], function (exports, _moduleName) {\\n\\t\\t\\\"use strict\\\";\\n\\n\\t\\tObject.defineProperty(exports, \\\"__esModule\\\", {\\n\\t\\t\\t\\tvalue: true\\n\\t\\t});\\n\\n\\t\\tvar _moduleName2 = _interopRequireDefault(_moduleName);\\n\\n\\t\\tfunction _interopRequireDefault(obj) {\\n\\t\\t\\t\\treturn obj && obj.__esModule ? obj : {\\n\\t\\t\\t\\t\\t\\tdefault: obj\\n\\t\\t\\t\\t};\\n\\t\\t}\\n\\n\\t\\texports.default = 42;\\n});\""`;
